django/core/management/commands/migrate.py

```diff
@@ -67,14 +67,15 @@ def handle(self, *args, **options):
             warnings.warn(
                 "The 'migrate --list' command is deprecated. Use 'showmigrations' instead.",
                 RemovedInDjango20Warning, stacklevel=2)
+            self.stdout.ending = None
```

Comment:
Tim pointed out to me that you are working on https://github.com/django/django/pull/3467 . I don't know if this interferes with #24068.

If your patch goes into 1.8 and makes the changes here and in #24068 obsolete, I'm not sure we even need to merge this patch and can close the ticket as duplicate.